cli/aws_orbit/remote_files/helm.py

```diff
@@ -131,6 +131,14 @@ def uninstall_chart(name: str) -> None:
         _logger.error(e)
 
 
+def uninstall_chart_in_namespace(name: str, namespace: str) -> None:
+    try:
+        _logger.debug("Uninstalling %s", name)
+        sh.run(f"helm uninstall --debug --namespace {namespace} {name}")
+    except exceptions.FailedShellCommand as e:
+        _logger.error(e)
+
+
```

Comment:
good catch , I also added a namespace to the uninstall on the kf branch

Reply:
I think I should remove the other one and all calls to it....without the namespace, the uninstall fails as nothing is installed in default...

Reply:
I take that back...anything installed in env will be default, so this method may be needed.  But the team plugins should call the method with the namespace...